<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14049" started-at="2018-06-06T17:53:48Z" finished-at="2018-06-06T17:54:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="14049" started-at="2018-06-06T17:53:48Z" finished-at="2018-06-06T17:54:02Z">
      <class name="com.testcases.LoginPagetest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.testcases.LoginPagetest@61832929]" name="setup" is-config="true" duration-ms="9785" started-at="2018-06-06T23:23:48Z" finished-at="2018-06-06T23:23:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="imgtest()[pri:0, instance:com.testcases.LoginPagetest@61832929]" name="imgtest" duration-ms="362" started-at="2018-06-06T23:23:58Z" finished-at="2018-06-06T23:23:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- imgtest -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:com.testcases.LoginPagetest@61832929]" name="Teardown" is-config="true" duration-ms="606" started-at="2018-06-06T23:23:58Z" finished-at="2018-06-06T23:23:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.testcases.LoginPagetest@61832929]" name="setup" is-config="true" duration-ms="2666" started-at="2018-06-06T23:23:59Z" finished-at="2018-06-06T23:24:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="titleverification()[pri:0, instance:com.testcases.LoginPagetest@61832929]" name="titleverification" duration-ms="50" started-at="2018-06-06T23:24:02Z" finished-at="2018-06-06T23:24:02Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Free CRM for Small Businesses - FreeCRM.com] but found [Free CRM software in the cloud powers sales and customer service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Free CRM for Small Businesses - FreeCRM.com] but found [Free CRM software in the cloud powers sales and customer service]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.testcases.LoginPagetest.titleverification(LoginPagetest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleverification -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:com.testcases.LoginPagetest@61832929]" name="Teardown" is-config="true" duration-ms="563" started-at="2018-06-06T23:24:02Z" finished-at="2018-06-06T23:24:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- com.testcases.LoginPagetest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
